src/pages/ScreeningsPage.vue

```diff
@@ -1,11 +1,156 @@
 <template>
-  <div></div>
+  <div>
+    <h1>Screenings</h1>
+    <h2>{{ selectedDay }}</h2>
+    <div>
+      <button
+        class="day-tabs"
+        v-for="day in daysList"
+        :key="day.id"
+        :data-date="day.date.toJSON()"
+        @click="setSelectedDay"
```

Comment:
instead of encoding the date to data attribute you can simply pass it as an argument to the method
```suggestion
        @click="setSelectedDay(day.date)"
```